packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -0,0 +1,146 @@
+import {Component, Element, h, Prop, State, VNode} from '@stencil/core';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {BaseFacet} from '../facet-common';
+import {
+  buildFacet,
+  buildSearchStatus,
+  Facet,
+  FacetOptions,
+  FacetSortCriterion,
+  FacetState,
+  FacetValue,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {registerFacetToStore} from '../../../utils/store';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {FacetValueLabelHighlight} from '../facet-value-label-highlight/facet-value-label-highlight';
+import {FacetValueBox} from '../facet-value-box/facet-value-box';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-segmented-facet',
+  styleUrl: 'atomic-segmented-facet.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacet
+  implements InitializableComponent, BaseFacet<Facet, FacetState>
+{
+  @InitializeBindings() public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @BindStateToController('facet')
+  @State()
+  public facetState!: FacetState;
+  public facet!: Facet;
+  @Element() private host!: HTMLElement;
+  @State() public error!: Error;
+
+  /**
+   * Specifies a unique identifier for the facet.
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  @Prop({reflect: true}) public field!: string;
+  /**
+   * The non-localized label for the facet.
```

Comment:
We should probably also add some jsdoc for the `label` of every facet which mentions that this also gets used by the `atomic-breadbox` component

Reply:
Will do